mlflow/ml-package-versions.yml

```diff
@@ -567,6 +567,7 @@ transformers:
       # prior to 4.30.0 are incompatible with this breaking change in Keras.
       # See: https://github.com/huggingface/transformers/issues/23352 for discussion
       "< 4.30.0": ["tensorflow<2.13.0"]
+      "== 4.34.1": ["accelerate<0.32.0"]
```

Comment:
- `pip install transformers==4.34.1 datasets` install `huggingface-hub-0.17.3`
- `accelerate==0.32.0` (released yesterday) imports `split_torch_state_dict_into_shards` from `huggingface_hub` and fails.